src/sdk/metadata.txt

```diff
@@ -1,3 +0,0 @@
-git@github.com:arduino-libraries/azure-iot-sdks.git
-arduino
-2bcecebd1ebbb0129db503c01ae15431e92a66e4
```

Comment:
Something like this meta data would be nice to have, to track which ```azure-iot-sdks``` version the library ships with.